tests/integration/test_eoc_journal.py

```diff
@@ -115,6 +163,52 @@ def test_pb_answer_blocks_listed_in_edit_view(self):
             label = item.find_element_by_css_selector('label')
             self.assertEqual(label.text, title)
 
+    def test_pb_answers_listed_in_student_view(self):
+        selected_block_ids = default_pb_answer_block_ids[1:]
+        self.configure_block(selected_pb_answer_blocks=selected_block_ids)
+
+        element = self.get_element_for_selected_pb_answers()
+
+        items = element.find_elements_by_css_selector('.eoc-selected-answers-section')
+        self.assertEqual(len(items), 1)
+        
+        section_name = items[0].find_elements_by_css_selector('h2')[0]
```

Comment:
You can use `find_element_by_css_selector('h2')` when you are only interested in the first element that matches.